lib/github-release.rb

```diff
@@ -119,7 +120,7 @@ def create_release(tag)
 
 		api.create_release(repo_name, tag, :name => name, :body => body)
 
-		puts " done!"
+		puts "done!"
```

Comment:
I, too, use Oxford commas, but not having a space after a pseudo-ellipsis just makes my skin crawl.  I also put two spaces after my periods, for which I receive much derision, so perhaps I just have an affinity for excessive whitespace.